.github/workflows/ci.yml

```diff
@@ -15,21 +15,29 @@ jobs:
     container: makeappdev/uselatex:latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Fetches all history for all branches and tags
```

Comment:
Why do we have to fetch everything?

Reply:
That is still from an older testing version. I think we can remove it.